src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = days * 40;
+
+  if (days >= 7) {
+    total = total - 50;
```

Comment:
You should also return value immediately in if statements (point 3 in [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md))